test/unit/test_text_to_speech_v1.py

```diff
@@ -1,248 +1,1224 @@
+    #-----------
+    def make_url(self, body):
+        endpoint = '/v1/customizations'
+        url = '{0}{1}'.format(base_url, endpoint)
+        return url
+
+    def add_mock_response(self, url, response):
+        responses.add(responses.GET,
+                      url,
+                      body=json.dumps(response),
+                      status=200,
+                      content_type='application/json')
+
+    def call_service(self, body):
+        service = TextToSpeechV1(authenticator=NoAuthAuthenticator(),)
+        service.set_service_url(base_url)
+        output = service.list_voice_models(**body)
+        return output
+
+    def construct_full_body(self):
+        body = dict()
+        body['language'] = "string1"
+        return body
+
+    def construct_required_body(self):
+        body = dict()
+        return body
+
+
+#-----------------------------------------------------------------------------
+# Test Class for update_voice_model
+#-----------------------------------------------------------------------------
+class TestUpdateVoiceModel():
+
+    #--------------------------------------------------------
+    # Test 1: Send fake data and check response
+    #--------------------------------------------------------
+    @responses.activate
+    def test_update_voice_model_response(self):
+        body = self.construct_full_body()
+        response = fake_response__json
+        send_request(self, body, response)
+        assert len(responses.calls) == 1
+
+    #--------------------------------------------------------
+    # Test 2: Send only required fake data and check response
+    #--------------------------------------------------------
+    @responses.activate
+    def test_update_voice_model_required_response(self):
+        # Check response with required params
+        body = self.construct_required_body()
+        response = fake_response__json
+        send_request(self, body, response)
+        assert len(responses.calls) == 1
+
+    #--------------------------------------------------------
+    # Test 3: Send empty data and check response
+    #--------------------------------------------------------
+    @responses.activate
+    def test_update_voice_model_empty(self):
+        check_empty_required_params(self, fake_response__json)
+        check_missing_required_params(self)
+        assert len(responses.calls) == 0
+
+    #-----------
+    #- Helpers -
+    #-----------
+    def make_url(self, body):
+        endpoint = '/v1/customizations/{0}'.format(body['customization_id'])
+        url = '{0}{1}'.format(base_url, endpoint)
+        return url
+
+    def add_mock_response(self, url, response):
+        responses.add(responses.POST,
+                      url,
+                      body=json.dumps(response),
+                      status=200,
+                      content_type='')
+
+    def call_service(self, body):
+        service = TextToSpeechV1(authenticator=NoAuthAuthenticator(),)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['TextToSpeechV1' may be undefined, or defined from star imports: ibm_watson.text_to_speech_v1](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4835642)